apps/web/app/components/mock-mode-toggle.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+
+import { Circle, styled } from 'leather-styles/jsx';
+import { TARGET, getLeatherMockMode, setLocalStorageMockMode } from '~/constants/environment';
+
+import { Flag } from '@leather.io/ui';
+import { delay } from '@leather.io/utils';
+
+import { BasicHoverCard } from './basic-hover-card';
+
+export function MockModeToggle() {
+  const [isMockMode, setIsMockMode] = useState(getLeatherMockMode());
+
+  useEffect(() => {
+    setIsMockMode(getLeatherMockMode());
```

Comment:
Do we have a `useOnMount()` in web app or did something change I should be aware of with empty dependencies?